tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -288,7 +303,7 @@ def run_conv_with_split(
                 torch_bias_zeroes_tensor, weights_dtype if weights_dtype != ttnn.bfloat8_b else ttnn.float32
             )
         torch_input_tensor = torch.permute(split_input_tensors[i], (0, 2, 3, 1))
-        tt_input_tensor = ttnn.from_torch(torch_input_tensor, ttnn.bfloat16)
+        tt_input_tensor = ttnn.from_torch(torch_input_tensor, ttnn.bfloat16, device=device)
```

Comment:
why do we push tensor to device here?
In run_conv we we don't push the tensor to device before the conv2d starts?

Reply:
This was a mistake, this `run_conv_with_split` should not be modified. Will fix.